scripts/test_catalyst.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# USAGE: test_catalyst.sh pod
+#
+# Builds and run tests for Catalyst since it's not yet supported by `pod lib lint`
+# The second argument should be "build" or "test". "Test" indicates both build and test.
+
+set -x
```

Comment:
It would be helpful to perform some argument validation in the script. As it stands if you just run this it's going to fail trying to find `.podspec`.

Also you should either explicitly check errors or opt into bash's `errexit` mode. You also probably want to enable `pipefail` mode so that the pipe to `xcpretty` does not mask an `xcodebuild` error. The easiest way to do both is to add this at the top:

```bash
set -euo pipefail
```

Finally, referring to `$1` and `$2` throughout isn't very readable. As part of your validation please pull out some variables like `pod` and `build_mode` or something like that.